bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/action/DoorbirdActions.java

```diff
@@ -54,8 +54,10 @@ public void setThingHandler(@Nullable ThingHandler handler) {
     @RuleAction(label = "restart the Doorbird", description = "Restarts the Doorbird device.")
     public void restart() {
         logger.debug("Doorbird action 'restart' called");
+        DoorbellHandler handler = this.handler;
```

Comment:
Typically, I prefer to use something like the following, as it avoid any potential naming confusion.

```
DoorbellHandler localHandler = handler;
```
Then use `localHandler` and `handler` in the remaining statements, where appropriate.

But this is a matter of personal preference, so if maintainer is ok with your changes...

Reply:
Agree, some while ago i had the same point of view, but i adapted to @jlaur preferences ;-) so i thinkt it is all good.

Reply:
I agree that it's a matter of personal preference, and I also agree with @lsiepel that it's my personal preference. 😄 Therefore both versions are of course accepted from a maintainer standpoint.

The reason I prefer explicit `this.` is because I don't like to come up with alternate names (i.e. prefixed by "local") when there is a way to **keep** the name and at the same time explicitly make the scope obvious for both references (local declaration is obvious and `this.` is also obviously an instance variable). An example where the alternate naming is a bit in the way to me is when the variable name is `host`. With that convention of prefixing by `local` it becomes `localHost`:
```java
String localHost = host;
```

This avoids confusion with "localhost" as a concept and also makes it obvious that right side is an instance variable without checking other code (i.e. just from reading this single line out of context):
```java
String host = this.host;
```

Just sharing my reasons, and feel free to prefer whatever you like. 😉